src/logging/oxen_logger.cpp

```diff
@@ -82,7 +82,21 @@ namespace oxen::logging
   {
     log::reset_level(log_level);
     log::add_sink(log::Type::Print, "stdout");
+    set_file_sink(log_location);
+    set_additional_log_categories(log_level);
+  }
+
+  void
+  init_file_logging(const std::string& log_location, log::Level log_level)
+  {
+    log::reset_level(log_level);
+    set_file_sink(log_location);
+    set_additional_log_categories(log_level);
+  }
```

Comment:
Might be nicer just to add a `bool stdout = true` parameter to `init()` rather than duplicating; aside from that one line these two functions are identical.